### DISCUSSION TITLE
Some important things I would like to see in AzuraCast

### DISCUSSION THREAD

**Discussion by greatdevvie** [AzuraCast/AzuraCast#6279] [Ideas]
Good morning.
I'm using Azuracast for a half of a year. It's a cool product, but I would like to see such things (are below) in Azuracast.
1) Flexible title for each mountpoint. It would be great to change a title of a mountpoint manually (if I want to).
2) Hiding mountpoint (out of statistics). It would be great if you make the possibility of hiding the mountpoint. For example: you have separate mountpoint for your bots and you need to remove them out of your statistics sight. You just make "hidden" mountpoint and the problem is solved.
Thank you.

**Comment by BusterNeece** [comment]
Hello,

You can submit feature requests for AzuraCast over at https://features.azuracast.com/

Thank you!
